src/Razor/src/Microsoft.VisualStudio.RazorExtension/Microsoft.VisualStudio.RazorExtension.csproj

```diff
@@ -53,6 +53,10 @@
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
       <IncludeInVSIX>true</IncludeInVSIX>
     </Content>
+    <Content Include="razorLanguageServiceCore64.servicehub.service.json">
```

Comment:
Do we plan to remove the `<Content>` block above this at a later time?

Reply:
From what I understand the plan is to support everything for a time being and then once everything is moved over to eventually remove the above content block.